src/analysis/vector/geometry_checker/qgsgeometrycheckfactory.h

```diff
@@ -67,22 +68,22 @@ class QgsGeometryCheckFactoryT : public QgsGeometryCheckFactory
 
     QString description() const override
     {
-      return static_cast<T *>( nullptr )->description();
+      return static_cast<T>( 0 ).description();
     }
 
     QString id() const override
     {
-      return static_cast<T *>( nullptr )->id();
+      return static_cast<T>( 0 ).id();
```

Comment:
@m-kuhn without this, it crashes when running

Reply:
Interesting. Let's see if it compiles for me now still...